pocs/focuser/focuslynx.py

```diff
@@ -271,8 +272,8 @@ def _initialise(self):
         self.logger.info('{} initialised'.format(self))
 
     def _get_hub_info(self):
-        self._hub_info = self._send_comman(command_str='<FHGETHUBINFO>',
-                                           expected_reply='HUB INFO')
+        self._hub_info = self._send_command(command_str='<FHGETHUBINFO>',
```

Comment:
Oh, that's a good one to find! I wonder why pyflakes or one of our other linters didn't catch that.

Reply:
I thin it doesn't (can't?) check things that are class methods, which is a little unfortunate.